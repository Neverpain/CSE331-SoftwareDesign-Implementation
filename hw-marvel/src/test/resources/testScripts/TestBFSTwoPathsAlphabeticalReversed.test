# Tests the findPath method from a node, n1, to another node, n4, that has two same
# length paths and prints out the most alphabetical path, edges added in reverse

CreateGraph g

AddNode g n1
AddNode g n2

AddEdge g n1 n2 B
AddEdge g n1 n2 A

FindPath g n1 n2