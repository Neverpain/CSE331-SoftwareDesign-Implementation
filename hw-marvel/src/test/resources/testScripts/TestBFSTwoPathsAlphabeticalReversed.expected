# Tests the findPath method from a node, n1, to another node, n4, that has two same
# length paths and prints out the most alphabetical path, edges added in reverse

created graph g

added node n1 to g
added node n2 to g

added edge B from n1 to n2 in g
added edge A from n1 to n2 in g

path from n1 to n2:
n1 to n2 via A